src/OpenTelemetry/Logs/OpenTelemetryLoggerProvider.cs

```diff
@@ -173,6 +178,7 @@ public ILogger CreateLogger(string categoryName)
         /// </remarks>
         public bool ForceFlush(int timeoutMilliseconds = Timeout.Infinite)
         {
+            OpenTelemetrySdkEventSource.Log.OpenTelemetryLoggerProviderEvent($"OpenTelemetryLoggerProvider.ForceFlush called with timeoutMilliseconds = {timeoutMilliseconds}.");
```

Comment:
Can probably avoid an allocation here by making it a dedicated event which accepts the timeout as an int parameter.